### PR TITLE
Hook up dashboard deployment and github notification

### PR DESCRIPTION
**Pull request by ChadNedzlek**
Hook up the last piece of the dashboard deployment as well
as fixing a few more lingering bugs with the status site
that were discovered when hooking up the notification

### CODE REVIEW

**Comment by ChadNedzlek** [comment]
@sunandabalu I think this should work with the new split deployment model. I have to rebuild some projects, but every stage of the deployment seems to sync the repository, so it should be fine...  Though we might want to disable cloning of the repository in most of the other stages.

@garath This should be the last part of the hookup

**Comment by sunandabalu** [comment]
> @sunandabalu I think this should work with the new split deployment model. I have to rebuild some projects, but every stage of the deployment seems to sync the repository, so it should be fine... Though we might want to disable cloning of the repository in most of the other stages.

The checkout in every stage is connected to the download task, its implicitly sync'ng the repo.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -218,6 +218,9 @@ stages:
               PathtoPublish: $(Build.ArtifactStagingDirectory)\tests\scenarios
               ArtifactName: ScenarioTests
               ArtifactType: Container
+          
+          - task: CopyFiles@2
```
Comment (alexperovich) [inline azure-pipelines.yml]: What files is this copying? It doesn't have any parameters.

File: src/Monitoring/Sdk/DeployPublisher.cs [hunk 2]
```diff
@@ -85,6 +86,7 @@ private async Task PostDatasourcesAsync()
 
                 Log.LogMessage(MessageImportance.Normal, "Posting datasource {0}...", name);
 
+                Debugger.Launch();
```
Comment (alexperovich) [inline src/Monitoring/Sdk/DeployPublisher.cs]: This probably should not be checked in.

File: azure-pipelines.yml (hunk 1 above)
Comment (ChadNedzlek) [reply azure-pipelines.yml]: I... didn't mean to change this file at all. i'll revert it

**Pull request merged**